src/style.css

```
@@ -0,0 +1,28 @@
+* {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  align-items: center;
+  flex-grow: 0;
+}
+
+.stars__star {
+  height: 12px;
+  width: 12px;
+  background-image: url(images/star.svg);
+  background-repeat: no-repeat;
+  margin: 2px;
+  padding: 0 2px;
+}
+
+.stars--1 > :first-child,
+.stars--2 > div:nth-child(-n+2),
```

```suggestion
.stars--2 .stars__star:nth-child(-n+2),
```
Можна використати таку комбінацію